packages/dapp-high-roller/src/components/app-root/app-root.tsx

```diff
@@ -8,6 +8,9 @@ import { cf, Node } from "../../data/types";
 declare var NodeProvider;
 declare var cf;
 
+const AUTH_TOKEN =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY4NzI5N2JjLTgwMTQtNGM4Mi04Y2VlLTNiN2NhN2RiMDlkNCIsInVzZXJuYW1lIjoiQWxvbnNraTIiLCJhZGRyZXNzIjoiMHhjNjBiOTAyM2JiOGRjMTUzYjQwNDY5NzczMjhjZTc5YWYxMmE3N2UwIiwiZW1haWwiOiJhbG9uMkBleGFtcGxlLmNvbSIsIm11bHRpc2lnQWRkcmVzcyI6IjB4OTQ5OWFjNUE2NmMzNjQ0N2U1MzVkMjUyYzA0OTMwNEQ4MDk2MUNFRCIsImlhdCI6MTU0NzQwNDUzNSwiZXhwIjoxNTc4OTYyMTM1fQ.innx0udzB2fj2_ZWYGugDJlz-Wb-qSckjqjyIBBTyS4";
```

Comment:
this shouldn't be necessary anymore as we posted a postMessage API for retrieving a token

Reply:
Deleted it